CRM/Upgrade/Incremental/php/FiveSixtyTwo.php

```diff
@@ -23,6 +23,19 @@ class CRM_Upgrade_Incremental_php_FiveSixtyTwo extends CRM_Upgrade_Incremental_B
 
   public function setPreUpgradeMessage(&$preUpgradeMessage, $rev, $currentVer = NULL) {
     if ($rev == '5.62.alpha1') {
+
```

Comment:
Of course when you originally wrote this it was alpha1, now it's beta1.
```suggestion
if ($rev == '5.62.beta1') {
```

Reply:
Confused? I still see 5.62.alpha1 here -> https://github.com/civicrm/civicrm-core/blob/625954e854d6ec2c34f6c40e52d71cf69670f7b5/CRM/Upgrade/Incremental/php/FiveSixtyTwo.php#LL25C18-L25C29
and 
https://github.com/civicrm/civicrm-core/blob/master/CRM/Upgrade/Incremental/php/FiveSixtyTwo.php#LL25C18-L25C29

Or do I need to account for the fact that we'll be making a new revisions here? In which case do I just add my upgrade logic in a different conditional to the existing?

Reply:
Oh right, sorry I didn't see there was already other stuff in the function. You'll need to add it to move it to an elseif clause, `elseif ($rev == '5.62.beta1') {`